src/main/java/org/jenkinsci/main/modules/sshd/CLICommandAdapter.java

```diff
@@ -21,22 +20,24 @@
     public Command create(CommandLine commandLine) {
         String cmd = commandLine.get(0);
         final CLICommand c = CLICommand.clone(cmd);
-        if (c==null)        return null;    // no such command
+        if (c == null) {
+            return null;    // no such command
+        }
```

Comment:
Please avoid gratuitous reformatting.

Reply:
> Please avoid gratuitous reformatting.

I will revert the change but this code is not even formatted